web/src/components/moreButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function MoreButton({ text, isDisabled, isLeft, isRight }) {
+    const disabledStyle = 'bg-brand-3';
+    const rightStyle = 'bottom-[-10%] right-8';
+    const leftStyle = 'top-[104%] left-[30px]';
+
+    const finalStyles = `absolute bg-black border-2 border-current rounded-bl-lg text-white text-center w-24 ${
+        isDisabled ? disabledStyle : ''
+    } ${isLeft ? leftStyle : ''}${isRight ? rightStyle : ''}`;
```

Comment:
Remove `bg-black` from `finalStyles`. 

Update your ternary to be `${isDabled ? disabledStyle : 'bg-black'`